CHANGE_LOG

```diff
@@ -1,3 +1,34 @@
+
+Version 0.21.0
+--------------
+
+This release introduces support for AMD's Heterogeneous System Architecture,
+which allows running the same code both on the CPU and the GPU.
```

Comment:
"which allows memory to be shared directly between the CPU and GPU."  The same code will not run on both CPU and GPU.